### ISSUE TITLE
Investigate aria-role best practices.

### ISSUE THREAD

**Issue by Mike-Heneghan** [Mike-Heneghan/ALISS#107]
Currently, some HTML elements are not used for their intended function. Semantically buttons should be used but anchor tags can enable behaviour without additional JS. 

Need to: 

- [x] Investigate best practices.
- [x] Implement best practices.

**Comment by Mike-Heneghan** [comment]
Good article on anchors vs buttons. 

https://bitsofco.de/anchors-vs-buttons/

**Comment by Mike-Heneghan** [comment]
https://medium.com/simple-human/but-sometimes-links-look-like-buttons-and-buttons-look-like-links-9b371c57b3d2

**Comment by Mike-Heneghan** [comment]
https://marcysutton.com/links-vs-buttons-in-modern-web-applications

**Comment by Mike-Heneghan** [comment]
https://davidwalsh.name/html5-buttons#targetText=The%20Difference%20Between%20Anchors%2C%20Inputs%20and%20Buttons&targetText=One%20of%20the%20things%20I,the%20third%20is%20an%20input.

**Comment by Mike-Heneghan** [comment]
In general, I think out buttons and some of our anchors are styled the same. There aren't any visual differences between them. 

Although the anchor tags generally "navigate the user to a page or resource" and the buttons (and inputs) are used for forms and behaviour which doesn't navigate elsewhere.

For example: 

<img width="767" alt="Screenshot 2019-09-19 at 15 00 59" src="https://user-images.githubusercontent.com/36415632/65251551-6cdd6b80-daef-11e9-8e56-21a37e031b29.png">

Some places these rules do not apply are the actions on the service detail pages etc. 

<img width="498" alt="Screenshot 2019-09-19 at 15 10 31" src="https://user-images.githubusercontent.com/36415632/65251678-a7df9f00-daef-11e9-81af-319788a89202.png">

They are all styled as links although some navigate away and others open modals or reveal input fields. 

Might be an opportunity to use the `role` attribute.

**Comment by Mike-Heneghan** [comment]
https://www.w3.org/TR/html-aria/#allowed-aria-roles-states-and-properties

**Comment by digitalWestie** [comment]
Should links that open modals also navigate to them?
